shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -79,6 +79,16 @@ + (Class)layerClass {
   if ([self.layer isKindOfClass:[CAEAGLLayer class]]) {
     fml::scoped_nsobject<CAEAGLLayer> eagl_layer(
         reinterpret_cast<CAEAGLLayer*>([self.layer retain]));
+    if ([[[NSBundle mainBundle] objectForInfoDictionaryKey:@(shell::kEmbeddedViewsPreview)]
```

Comment:
Please put this check and similar others in one spot so they can be updated in one go.

Reply:
Sent #6813 will update this call site in whichever of the PRs that lands second.